reviews/2021-Kyverno-annual.md

```diff
@@ -0,0 +1,132 @@
+* [CNF WG tests](https://github.com/cncf/cnf-testsuite)
+
+**Organizations**
+* [Ohio Supercomputer Center](https://www.osc.edu/)
+* [VSHN](https://www.vshn.ch/en/)
+* [Coinbase](https://www.coinbase.com/)
+* [Mandiant](https://www.mandiant.com/)
+* [Giant Swarm](https://www.giantswarm.io/)
+* [Vodafone Group Plc](https://www.vodafone.com/)
+* [Deutsche Telekom](https://www.telekom.com/en)
+
+A list of companies providing commercial services and solutions for Kyverno is available [here](https://kyverno.io/support/).
+
+## Project Goals
+
+> How has the project performed against its goals since the last review? (We won't penalize you if your goals changed for good reasons.)
+
+The main goals of the project have been in the following categories:
+
+* **Complex policy support**: Several new features such as [JMESPath support](https://kyverno.io/docs/writing-policies/variables/), [API Server Lookups](https://kyverno.io/docs/writing-policies/external-data-sources/#variables-from-kubernetes-api-server-calls), and [foreach](https://kyverno.io/docs/writing-policies/validate/#foreach) have been introduced to allow declarative processing of complex policy logic.
+
+* **Availability**: Kyverno has added the ability to support multiple replicas for fault-tolerance and increased availability in Release 1.4.0 (June 2021).
+
+* **Scalability**: Several fixes and improvenents have been made to reduce memory usage and handle large clusters. This remains an on-going area of test and improvement.
+
+* **Security**: Kyverno participated in the CNCF TAG Security [Security Pals](https://github.com/cncf/tag-security/issues/554) program. As a result, Kyverno has adopted recommended security best practices. The [Security](https://main.kyverno.io/docs/security/) section in the Kyverno documentation lists these and acts as a security guide for adopters.
+
+* **Community and Awareness**: Kyverno has grown from initially having maintainers from a single company, to maintainers from multiple organizations. The community has also grown significantly as reflected by [DevStats](https://kyverno.devstats.cncf.io). While Kyverno awareness has grown, more work is needed to make the community fully aware of all of its features and capabilities.
+
+> What are the current goals of the project? For example, are you working on major new features? Or are you concentrating on adoption or documentation?
+
+1. **Scalability**: Kyverno is increasingly being used in large production clusters. We aim to dramatically improve scalability in the subsequent releases.
+
+2. **Security**: In addition to current security processes, a goal for Kyverno is to improve software supply chain security and achieve [SLSA Level 4](https://slsa.dev/levels).
+
+3. **Adoption**: We aim to continue growing Kyverno adoption. A survey conducted by Nirmata at KubeCon US 2021 showed Kyverno adoption is currently around 8%. Our goal is to exceed 30% adoption by Q4 2022.
```

Comment:
is there a plan for this that is public? (what steps/when etc)

Reply:
Here is the tracking issue for the [SLSA Level 3](https://github.com/kyverno/kyverno/issues/3119) work which is planned as an LFX Mentorship program that just started.